packages/systemd-252/systemd-252.spec

```
@@ -164,11 +170,40 @@ Provides: %{_cross_os}systemd-networkd = %{package_priority_epoch}:
 %package resolved
 Summary: Files for resolved
 Requires: %{name}
+Requires: %{name}-resolved(dbus-config)
 Provides: %{_cross_os}systemd-resolved = %{package_priority_epoch}:
 
 %description resolved
 %{summary}.
 
+%package dbus-launcher-config
+Summary: Dbus launcher configuration files
+Requires: %{name}
+Provides: %{name}(dbus-config) = 1:
+Provides: %{name}-resolved(dbus-config) = 1:
+Provides: %{name}-networkd(dbus-config) = 1:
+Conflicts: (%{_cross_os}whippet or %{name}-whippet-config)
+Conflicts: %{name}(dbus-config)
+Conflicts: %{name}-resolved(dbus-config)
+Conflicts: %{name}-networkd(dbus-config)
+
+%description dbus-launcher-config
+%{summary}.
+
+%package whippet-config
+Summary: Whippet configuration files
+Requires: %{name}
+Provides: %{name}(dbus-config) = 1:
+Provides: %{name}-resolved(dbus-config) = 0:
+Provides: %{name}-networkd(dbus-config) = 0:
+Conflicts: (%{_cross_os}dbus-broker-launcher or %{name}-dbus-launcher-config)
```

nit: this conflict is doing too much work. It also feels like a bug that `whippet-config` doesn't require `whippet` and `dbus-launcher-config` doesn't require `dbus-broker-launcher`.

You can just ensure that `dbus-broker-launcher` and `whippet`  - the respective dependencies - conflict with each other, and that these config packages conflict with each other, and that each depends on its launcher.
```suggestion
Requires: %{_cross_os}whippet
Conflicts: %{name}-dbus-launcher-config
```